SmartDeviceLink/SDLProxy.h

```diff
@@ -42,7 +42,7 @@ __deprecated_msg("Use SDLManager instead")
 - (void)sendRPCRequest:(SDLRPCMessage *)msg __deprecated_msg("use -sendRPC: instead");
 
 - (void)handleRPCDictionary:(NSDictionary *)dictionary;
-- (void)handleRpcMessage:(NSDictionary *)msg __deprecated_msg("use -handleRPCDictionary: instead");
+- (void)handleRPCMessage:(NSDictionary *)msg __deprecated_msg("use -handleRPCDictionary: instead");
```

Comment:
This was a mis-commit when using regex to modify the 1,000,000 files involved in this PR.